src/applications/mhv-landing-page/components/LandingPage.jsx

```diff
@@ -5,57 +5,51 @@ import { renderMHVDowntime } from '@department-of-veterans-affairs/mhv/exports';
 import DowntimeNotification, {
   externalServices,
 } from '~/platform/monitoring/DowntimeNotification';
-import { isLOA1 } from '~/platform/user/selectors';
-import { signInServiceName } from '~/platform/user/authentication/selectors';
 import { SERVICE_PROVIDERS } from '~/platform/user/authentication/constants';
 import IdentityNotVerified from '~/platform/user/authorization/components/IdentityNotVerified';
 // eslint-disable-next-line import/no-named-default
 import { default as recordEventFn } from '~/platform/monitoring/record-event';
 
 import CardLayout from './CardLayout';
-import NoHealthAlert from './NoHealthAlert';
+import UnregisteredAlert from './UnregisteredAlert';
 import HeaderLayoutV1 from './HeaderLayoutV1';
 import HeaderLayout from './HeaderLayout';
 import HubLinks from './HubLinks';
 import NewsletterSignup from './NewsletterSignup';
 import WelcomeContainer from '../containers/WelcomeContainer';
-import { hasHealthData, personalizationEnabled } from '../selectors';
+import {
+  isLOA1,
+  personalizationEnabled,
+  signInServiceName,
+} from '../selectors';
 
-const LandingPage = ({ data = {}, recordEvent = recordEventFn }) => {
+const LandingPage = ({ data = {}, recordEvent = recordEventFn, vaPatient }) => {
   const { cards = [], hubs = [] } = data;
-  const isUnverified = useSelector(isLOA1);
-  const hasHealth = useSelector(hasHealthData);
+  const loa1 = useSelector(isLOA1);
```

Comment:
Instead of renaming the selected prop from state, use the prop name from state as the variable name.